packages/cli/src/commands/bisect/index.ts

```diff
@@ -370,6 +366,14 @@ export default async function main(client: Client): Promise<number> {
   return 0;
 }
 
+function hasScheme(url: string): Boolean {
+  return url.startsWith('http://') || url.startsWith('https://');
+}
+
+function normalizeURL(url: string): string {
+  return hasScheme(url) ? url : `https://${url}`;
+}
```

Comment:
For the purposes of testing this PR, I would recommend moving this into a file in `src/utils/commands/bisect` dir and adding some unit tests with various input values.

Reply:
👍 Just added tests

Reply:
Awesome. Thanks!